requirements.txt

```diff
@@ -2,4 +2,4 @@ pennylane>=0.32
 pennylane-catalyst>=0.6
 pyqrack>=0.13.0
 numpy~=1.16
-scikit-build
+scikit-build>=0.1.0
```

Comment:
Do we need to specify these requirements in both the setup and requirements? I believe there's a way to define them in one spot and then use them in both.

Reply:
There is, but many projects do it either way. For now, I prefer this to piping in the requirements.txt.

Reply:
I looked into it a bit, for due diligence: `setup.py` needs installation dependencies; `requirements.txt` needs development build dependencies. Except in the case these two dependency lists are identical, perhaps they should be kept distinct.